WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/DeviceInfo.cs

```diff
@@ -398,6 +404,27 @@ public partial class Resolution
             [DataMember(Name = "Index")]
             public int Index { get; private set; }
         }
+
+        /// <summary>
+        /// Error information if a request fails.
+        /// </summary>
+        [DataContract]
+        public partial class ErrorInformation
```

Comment:
HttpErrorResponse in DevicePortalException.cs looks to be generalized enough that you could use it instead of this. It doesn't currently have a "Status" data member but you can add one.